myapp/app/controllers/tasks_controller.rb

```diff
@@ -3,14 +3,17 @@ def index
     @tasks = Task.order(due: :desc)
   end
 
-  def new; end
+  def new 
+    @new_task = Task.new
+  end
 
   def create
-    @new_task = Task.new(params.require(:task).permit(:title, :description))
-    @new_task.save
-
-    flash[:success] = 'New task was created successfully!'
-    redirect_to tasks_path
+    @new_task = Task.new(params.require(:task).permit(:title, :description, :due))
```

Comment:
It might be better to define this params part as a method so that line 29 can also use it.

```
private

def task_params
  params.require(:task).permit(:title, :description, :due)
end
```

Reply:
Thank you! Fixed it in 5626e84

Reply:
早い！w
ありがとうございます。他のところ見るので少々お待ちください 🙇